vmicore/src/lib/os/windows/KernelOffsets.cpp

```diff
@@ -45,7 +45,10 @@ namespace VmiCore::Windows
                                 .Right = vmiInterface->getKernelStructOffset("_RTL_BALANCED_NODE", "Right")},
             .fileObject = {.FileName = vmiInterface->getKernelStructOffset("_FILE_OBJECT", "FileName")},
             .section = {.controlArea = vmiInterface->getKernelStructOffset("_SECTION", "u1")},
-            .kprocess = {.DirectoryTableBase = vmiInterface->getKernelStructOffset("_KPROCESS", "DirectoryTableBase")},
+            .kprocess = {.kernelDirectoryTableBase =
```

Comment:
I suggest renaming `kernelDirectoryTableBase` to `directoryTableBase` across the whole code base since we cannot know in advance whether it's always just the kernel dtb. It depends on the circumstances.